pkgs/development/python-modules/torchbench/default.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  accelerate,
+  boto3,
+  beautifulsoup4,
+  distro,
+  iopath,
+  monkeytype,
+  numpy,
+  opencv4,
+  patch,
+  psutil,
+  py-cpuinfo,
+  pynvml,
+  pytest,
+  pytest-benchmark,
+  pyyaml,
+  requests,
+  submitit,
+  tabulate,
+  transformers,
+}:
+
+buildPythonPackage {
+  pname = "torchbench";
```

Comment:
`torchbench` is [another project](https://pypi.org/project/torchbench/). How about `torch-bench`?

Reply:
This is precisely that project https://pypi.org/project/torchbench/ that I would like to package here. How to make sure the project I see on Pypi is the one on Github?

Reply:
- [Pypi's `torchbench`](https://pypi.org/project/torchbench) is [paperswithcode/torchbench](https://github.com/paperswithcode/torchbench)
- [Pypi's `torchbenchmark`](https://pypi.org/project/torchbenchmark/) is [pytorch/benchmark](https://github.com/pytorch/benchmark)

Reply:
If you would like to package `torchbench` (the first one), you have to use `fetchPypi` as they unfortunarely [don't tag released on GitHub](https://github.com/paperswithcode/torchbench/tags).